getting_started/install_openshift.adoc

```diff
@@ -181,10 +203,16 @@ Follow the prompts and just hit enter when asked for pass phrase.
 
 An easy way to distribute your SSH keys is by using a `bash` loop:
 
+[source,terminal]
 ----
 $ for host in master.openshift.example.com \
     node.openshift.example.com; \
-    do ssh-copy-id -i ~/.ssh/id_rsa.pub $host; \
+    do ssh-copy-id -i ~/.ssh/id_rsa.pub $host;
+----
+
+.Example output
```

Comment:
@ahardin-rh I don't think that this change is right. The `done` is part of the above for loop above, so that should be added back to the same code block (and with the \).

Reply:
@bergerhoffer Good catch! Updated in https://github.com/openshift/openshift-docs/pull/25178 Thank you!